src/include/duckdb/main/client_config.hpp

```diff
@@ -35,6 +82,9 @@ struct ClientConfig {
 	//! The file to save query profiling information to, instead of printing it to the console
 	//! (empty = print to console)
 	string profiler_save_location;
+	//! The custom settings for the profiler
+	//! (empty = use the default settings)
+	TreeNodeSettings profiler_settings;
```

Comment:
Is this done on a per-client basis, or for the entire instance? I think if we set it with a pragma, then it has to be in the `DBConfig`?